geneticalg/geneticalg.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python
+
+
+import numpy as np
+
+
+def gen_chrom(max_seconds, video_list, video_lengths):
```

Comment:
having a doc string describing the parameters (and their expected types) would be really handy when you get back to use this function later on.
For me -- ATM I even have no clue what `chrom` stands for and what to expect from the returned value

Reply:
Done!